From 641c586eae8fabf345a36572636776157d799ebe Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Tue, 01 Feb 2022 14:56:52 +0200
Subject: [PATCH] [SW-69581] ppboot: gaudi: increase sram footprint size

In the following patches device specific headers will be
removed from common code, this slightly increase the sram footprint
of gaudi ppboot application.
The binary size remain within the required 20kbytes, and is
verified by the makefile post script.

Change-Id: Ida3b61deafc961474fd22381dfe2495e310d1dd2
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi/arm/common.h b/arch/gaudi/arm/common.h
index 60afdc2..67eb24f 100644
--- a/arch/gaudi/arm/common.h
+++ b/arch/gaudi/arm/common.h
@@ -10,7 +10,7 @@
 #include <address_space.h>
 #include <sizes.h>
 
-#define PPBOOT_SIZE			0x5000		/* 20KB */
+#define PPBOOT_SIZE			0x6000		/* 24KB */
 #define PREBOOT_SIZE			0x25000		/* 148KB */
 
 #define BOOT_INFO_BASE			HL_SP_SRAM_BASE
